cms/templates/course_outline.html

```diff
@@ -69,6 +69,11 @@ <h3 class="sr">${_("Page Actions")}</h3>
                     </a>
                 </li>
                 <li class="nav-item">
+                    <a href="#" class="button re-index" data-category="reindex" title="${_('Click to reindex current course')}">
```

Comment:
I think that saying "Click to..." is unnecessary. You can just say `title="${_('Reindex current course')}"`